cmd/clusterctl/cmd/config_repositories_test.go

```diff
@@ -211,6 +212,10 @@ var expectedOutputYaml = `- File: core_components.yaml
   Name: nested
   ProviderType: InfrastructureProvider
   URL: https://github.com/kubernetes-sigs/cluster-api-provider-nested/releases/latest/
+  - File: infrastructure-components.yaml
```

Comment:
```suggestion
- File: infrastructure-components.yaml
```
probably the test issue

Reply:
Fixed